pep-0520.txt

```diff
@@ -410,20 +410,14 @@ discovery is almost entirely an implementation detail.
 References
 ==========
 
-.. [impl] issue #24254
-   (https://bugs.python.org/issue24254)
-
-.. [nick_concern] Nick's concerns about mutability
-   (https://mail.python.org/pipermail/python-dev/2016-June/144883.html)
-
-.. [orig] original discussion
-   (https://mail.python.org/pipermail/python-ideas/2013-February/019690.html)
+* `Original discussion
+  <https://mail.python.org/pipermail/python-ideas/2013-February/019690.html>`__
 
-.. [followup1] follow-up 1
-   (https://mail.python.org/pipermail/python-dev/2013-June/127103.html)
+* `Follow-up 1
+  <https://mail.python.org/pipermail/python-dev/2013-June/127103.html>`__
 
-.. [followup2] follow-up 2
-   (https://mail.python.org/pipermail/python-dev/2015-May/140137.html)
+* `Follow-up 2
+  <https://mail.python.org/pipermail/python-dev/2015-May/140137.html>`__
```

Comment:
+1 on moving these links into a bulleted list.  SOP (standard operating procedure) at the time was to add references even if they weren't used.  Personally I prefer bulleted list list this for unreferenced links. 😄  So thanks for making this change!